using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -348,7 +406,7 @@ If your application uses more complicated URLs, such as Angular URLs, we recomme
 
 ### appendVisitorInfoForURL <a id="appendToUrl-js"></a>
 
-Appends Adobe visitor information to the query component of the given URL.
+This `appendVisitorInfoForURL()` API appends Adobe visitor information to the query component of the given URL.
```

Comment:
Reword so that it reads: "This API appends Adobe visitor information to the query component of the specified URL."